portfolio/src/main/java/com/google/sps/servlets/PostDataServlet.java

```diff
@@ -45,39 +45,26 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String email = getParameter(request, "email", "");
     String message = getParameter(request, "message", "");
 
-    // Create an Entity type Post.
-    Entity postEntity = new Entity("Post");
-    postEntity.setProperty("firstName", firstName);
-    postEntity.setProperty("lastName", lastName);
-    postEntity.setProperty("email", email);
-    postEntity.setProperty("message", message);
-
-    // Determine if message is appropriate for posting.
+    // Determine sentiment score using Google Natural Langugage API.
     Document doc =
         Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
     LanguageServiceClient languageService = LanguageServiceClient.create();
     Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
-    float sentimentScore = sentiment.getScore();
+    Double sentimentScore = (double) sentiment.getScore();
```

Comment:
You can use the primitive `double` here instead of `Double`

Reply:
Fixed!